### DISCUSSION TITLE
Any way to switch between CT and RGB modes without modifying values?

### DISCUSSION THREAD

**Discussion by Shituation** [arendst/Tasmota#16813] [Q&A]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

(Please, remember to close the issue when the problem has been addressed)

**Cannot see any commands to switch modes like in tuya bulbs. Tuya has the "mode_command" to switch between "colour" (RGB) and "white" (CT) but Tasmota doesn't. 
Is there any way to change modes without sending new values? (brightness, hue, etc etc), I've read the documentation but could not find anything related to this.**

**What I want is to be able to switch modes without altering any values.**

**Comment by sfromis** [comment]
Do not use Issues to ask questions, instead of having reasons to assume a bug.

**Reply by Shituation** [reply]
Putting pressure to users to use discord or other data mining platforms is not a good policy, you know?
If you forgot to add something so simple to your software as switching light modes, at least admit it and try to make it better instead of closing issues like a policeman.

I don't care anymore. Thanks.

**Reply by sfromis** [reply]
I made no pressure to go to Discord, just that opening an "issue" is not the right way, and the thread was moved to the appropriate place, Discussions.

**Reply by s-hadinger** [reply]
@Shituation I'm not sure we "forgot" to add a feature to the software since you are the first to ask for this feature. This feature is not implemented and I understand that it may be useful.

You care or you don't care?
